xc7/yosys/install_synth.tcl

```diff
@@ -0,0 +1,43 @@
+yosys -import
```

Comment:
Copying and pasting the xc7/synth.tcl is not accepted.  Please find a way to have one script

Reply:
fixed